instance/static/html/instance/index.html

```diff
@@ -58,12 +65,22 @@
               </li>
             </ul>
           </div>
-          <div ng-switch="selected.instance.status">
-            <button class="button reprovision" ng-switch-when="ready"
-                                               ng-click="provision(selected.instance)">
-              Reprovision
-            </button>
-            <img src="/static/img/running.png" tooltip-placement="bottom" tooltip="Provisioning in progress..." alt="Provisioning in progress..." ng-switch-default />
+          <div class="large-2 columns">
+            <div class="status-icon" ng-switch="selected.instance.status">
+              <button class="button reprovision" ng-switch-when="ready"
+                                                 ng-click="provision(selected.instance)">
+                Reprovision
+              </button>
+              <button class="button reprovision" ng-switch-when="error"
+                                                 ng-click="provision(selected.instance)">
+                Reprovision
+              </button>
```

Comment:
Similar remarks as above - I'd rather ensure that the progress gives an accurate representation of whther there are still pending actions in progress, and use that as the basis to decide if the user is allowed to start new actions.